forum/qa-plugin/q2a-badges-master/qa-badge-check.php

```diff
@@ -358,20 +358,17 @@ function answer_vote_down($event,$event_user,$params) {
 			if($event_user) $this->check_voter($event_user);
 		}
 
-		function check_voted($uid) {
-						// upvotes received
+                function check_voted($uid) {
+                    // upvotes received
 			
-			$votes = qa_db_read_one_assoc(
-				qa_db_query_sub(
-					'SELECT upvoteds FROM ^userpoints WHERE userid=#',
-					$uid
-				),
-				true
-			);
-			$badges = array('liked','loved','revered');
+                    $votesArray = qa_db_query_raw('SELECT upvoteds FROM qa_userpoints WHERE userid='.$uid.' LIMIT 1');
+                    foreach($votesArray as $vote) {
+                        $votes = $vote;
+                    }
+                    $badges = array('liked','loved','revered');

```

Comment:
Użyłbym `[ ]` zamiast `array()`

Reply:
To już kod twórców Q2A, nie ruszałem tego, ale zmienie

Reply:
Wracamy do kwestii która już była omawiana - zmieniając coś w danym pliku warto byłoby moim zdaniem ogólnie poprawiać jakość kodu w nim całym (PSR-2, nazewnictwo, zapis). Nie zawsze jest na to czas, więc powiedzmy że na to można przymknąć oko, ale jak coś poprawiamy tuż obok tego no to już bym to postarał się zmieniać.